sim_pipeline/gg_lens_pop.py

```diff
@@ -65,7 +75,21 @@ def get_num_lenses(self):
     def get_num_sources(self):
         return self._source_galaxies.galaxies_number()
 
-    def draw_population(self, testarea=4 * np.pi):
+    def get_num_sources_tested(self, testarea):
+        """
+         To show the mean of source galaxies needed to tested within the test area.
+         (later draw a normal distribution around the mean for the number of source galaxies tested);
+         num_sources_tested_mean/ testarea = num_sources/ f_sky;
+         testarea is in units off arcsec^2, f_sky is in units of deg^2. 1 deg^2 = 12960000 arcsec^2
+         num_source_range is in Poisson distribution with mean of num_sources_tested_mean.
+
+        """
+        num_sources = self._source_galaxies.galaxies_number()
+        num_sources_tested_mean = (testarea * num_sources) / (12960000 * self.f_sky)
```

Comment:
here you are combining two different things. One is the average source for the render area, and the other is drawing a realization of the expected distribution. I would put the poisson draw in the other function with the strong lensing realization instead of this one